### ISSUE TITLE
Write Documentation

### ISSUE THREAD

**Issue by kanderoo** [kanderoo/vim-tabs#3]
Things like Chord Completion in particular should be clearer than they are.

**Comment by Felixoid** [comment]
Hi. I recently started jumping around here and there to learn to play guitar and ended up here. It looks like the most complete vim plugin for the tabs.

Say, are you interested in the PRs to improve some things?

**Comment by kanderoo** [comment]
Absolutely, go for it! Honestly I don't really remember where I left off on this project, I'm not sure I left it in a very completed state. Feel free to open a PR if you have any ideas to make it better
